config/routes.rb

```diff
@@ -4,6 +4,9 @@
   post 'users/new', action: :create, controller: 'users'
   get 'users', action: :index, controller: 'users'
   delete 'users/:id', action: :destroy, controller: 'users'
+  get 'users/:id/edit', action: :edit, controller: 'users'
+  patch 'users/:id', action: :update, controller: 'users'
+  put 'users/:id', action: :update, controller: 'users'
```

Comment:
Are you sure that you need both? Usually people Just use one, The most popular ia put

Reply:
I am going to use only the PUT. thanks for the advice